src/store/bridge.ts

```diff
@@ -239,12 +256,30 @@ const mutations = {
   [types.SET_HISTORY_ITEM] (state, historyItem: BridgeHistory | null) {
     state.historyItem = historyItem
   },
-  [types.SEND_TRANSACTION_REQUEST] (state) {},
-  [types.SEND_TRANSACTION_SUCCESS] (state) {},
-  [types.SEND_TRANSACTION_FAILURE] (state) {},
-  [types.RECEIVE_TRANSACTION_REQUEST] (state) {},
-  [types.RECEIVE_TRANSACTION_SUCCESS] (state) {},
-  [types.RECEIVE_TRANSACTION_FAILURE] (state) {}
+  [types.SIGN_SORA_TRANSACTION_SORA_ETH_REQUEST] (state) {},
```

Comment:
In the future it is better to make this code more generic because we will add other bridges than ETH